pymbolic/primitives.py

```diff
@@ -270,16 +301,16 @@ class Expression:
         `PEP 634 <https://peps.python.org/pep-0634/>`__-style pattern matching
         is now supported when Pymbolic is used under Python 3.10.
 
-    .. attribute:: a
+    .. autoattribute:: a
```

Comment:
```suggestion
    .. autoproperty:: a
```
? Does this work if the return type is not documented?